src/module.c

```diff
@@ -12285,7 +12285,7 @@ int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loa
     }
 
     int dlopen_flags = RTLD_NOW | RTLD_LOCAL;
-#if (defined(__linux__) || defined(__FreeBSD__)) && !defined(__SANITIZE_ADDRESS__)
+#if (defined(__linux__) || defined(__FreeBSD__)) && !defined(__SANITIZE_ADDRESS__) && defined(__GLIBC__) && __has_include(<dlfcn.h>)
```

Comment:
FreeBSD doesn't GLIBC. I think this change disables DEEPBIND on FreeBSD.

Reply:
Maybe we should just check `__GLIBC__` instead of `__linux__`? The libc implementation is what defines dlopen.

```suggestion
#if (defined(__GLIBC__) || defined(__FreeBSD__)) && !defined(__SANITIZE_ADDRESS__) && __has_include(<dlfcn.h>)
```